api-gateway/src/main/resources/static/scripts/product-details-info/product-details-info.controller.js

```diff
@@ -6,16 +6,28 @@ angular.module('shopProductDetailsInfo')
         self.product = {}; // Initialize self.product
         var productId = $stateParams.productId;
 
-        $http.get('/api/v2/gateway/products/' + productId)
+        $http.get('/api/gateway/products/' + productId)
             .then(function (resp) {
                 // Handle the response data for the specific product
                 var product = resp.data;
                 //console.log("Product found:", product);
                 self.product = product; // Update the product data in your controller
+
+                //fetch image
+                $http.get('api/v2/gateway/images/' + product.imageId).then(function (imageResp){
```

Comment:
```suggestion
                $http.get('api/gateway/images/' + product.imageId).then(function (imageResp){
```
There is a V1 for images now

Reply:
thank you, controllers have been patched